eddymotion/tests/test_estimator.py

```diff
@@ -20,4 +20,54 @@
 #
 #     https://www.nipreps.org/community/licensing/
 #
-"""Unit tests exercising the estimator."""
\ No newline at end of file
+"""Unit tests exercising the estimator."""
+from tempfile import TemporaryDirectory
+from pkg_resources import resource_filename as pkg_fn
+from pathlib import Path
+
+import nitransforms as nt
+import numpy as np
+import random
+
+from nibabel.eulerangles import euler2mat
+from nibabel.affines import from_matvec
+from nipype.interfaces.ants.registration import Registration
+
+
+def test_ANTs_config_b0(pkg_datadir, tmpdir):
```

Comment:
This will test all combinations of the 6 parameters and their values.
```suggestion
@pytest.mark.parametrize("r_x", [0.0, 0.01, 0.1, 0.3])
@pytest.mark.parametrize("r_y", [0.0, 0.01, 0.1, 0.3])
@pytest.mark.parametrize("r_z", [0.0, 0.01, 0.1, 0.3])
@pytest.mark.parametrize("t_x", [0.0, 0.5, 1.0])
@pytest.mark.parametrize("t_y", [0.0, 0.5, 1.0])
@pytest.mark.parametrize("t_z", [0.0, 0.5, 1.0])
def test_ANTs_config_b0(pkg_datadir, tmpdir, r_x, r_y, r_z, t_x, t_y, t_z):
```